base/sysimg.jl

```diff
@@ -500,31 +500,52 @@ using Base
 # Ensure this file is also tracked
 pushfirst!(Base._included_files, (@__MODULE__, joinpath(@__DIR__, "sysimg.jl")))
 
+macro stdlib(name)
+    println(name)
```

Comment:
Is this `println` deliberate?

Reply:
It looks like vchuravy intended to write `function`, but instead wrote `macro`? :P

Jokes aside though, we don't want to define anything (variables, functions, macros, etc) in this namespace.

Reply:
Oh the original intend was to get a progress message. So the `println` was intended

Reply:
Printing would be great. I hate looking at `Docs.jl`°

Reply:
True. I think that once logging can be enabled from the command line, setting `env JULIA_DEBUG_LOADING=1` in the Makefile (as we could do on v0.6) would be a good way to do this in a generic way.